pypeit/scripts/sensfunc.py

```diff
@@ -111,13 +110,14 @@ def main(args):
                        "    [sensfunc]\n"
                        "         algorithm = IR\n"
                        "\n")
-        if args.flatfile is not None and args.sens_file is not None:
-            msgs.error("It is not possible to set --flatfile and simultaneously use a .sens "
+
+        if not args.use_flat and args.sens_file is not None:
```

Comment:
Thank you for adding this.
I think here you want `if args.use_flat and args.sens_file is not None:`
The idea is: if the `--use_flat` flag is not used, it means `args.use_flat` is `False` and in that case we use the default value for the instrument or the value in the `.sens` file. If the `--use_flat` flag  is used, `args.use_flat` is `True` and that would be the value used, which clashes with the `.sens` file (if provided), hence this error.

Reply:
Indeed -- good catch! I'll fix this and restart my dev-suite run. Thanks for catching that!